### PR TITLE
[FIX] simplify code and do not use pushEditOperations

### PR DESCRIPTION
**Pull request by CGNonofr**

### CODE REVIEW

File: src/MonacoEditor.tsx [hunk 1]
```diff
@@ -231,20 +231,9 @@ function MonacoEditor ({
...
-              range: model.getFullModelRange(),
-              text: fixedCode
-            }],
-            () => null
-          )
-          editor.pushUndoStop()
-        }
+        editor.pushUndoStop()
+        model.setValue(fixedCode)
+        editor.pushUndoStop()
```
Comment (samuel-olivier) [inline src/MonacoEditor.tsx:236]: what will it do in a read-only IDE ?

Comment (CGNonofr) [reply src/MonacoEditor.tsx:236]: Are you talking about the ` pushUndoStop`?

Comment (samuel-olivier) [reply src/MonacoEditor.tsx:236]: yes

Comment (CGNonofr) [reply src/MonacoEditor.tsx:236]: Probably nothing as it probable the content wasn't changed for a long time

=>  try in https://microsoft.github.io/monaco-editor/playground.html :
```javascript
const model = monaco.editor.createModel("function hello() {\n\talert('Hello world!');\n}", 'javascript')
const editor = monaco.editor.create(document.getElementById('container'), {
	model,
    readOnly: true
});

setTimeout(() => {
    editor.pushUndoStop()
    model.setValue('omg')
    editor.pushUndoStop()
}, 1000)
```

**Review by samuel-olivier — APPROVED** [review]

**Pull request merged**